beard_server/app.py

```diff
@@ -40,4 +40,7 @@
 logging.basicConfig(stream=sys.stderr)
 
 if __name__ == '__main__':
+    # Text endpoint sends back keys UTF-8.
+    app.config['JSON_AS_ASCII'] = False
```

Comment:
I think this is how it should be, non? I am not sure how they are doing in Invenio*